qa/crontab/teuthology-cronjobs

```diff
@@ -67,7 +67,7 @@ CEPH_QA_EMAIL="ceph-qa@ceph.io"
 05 04 * * 1,6 CEPH_BRANCH=master; MACHINE_NAME=smithi;    /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH -n 7 -m $MACHINE_NAME -s ceph-ansible -k distro -e $CEPH_QA_EMAIL
 ###  The suite below must run on bare-metal because it's performance suite and run 3 times to produce more data points
 57 03 * * 2,5 CEPH_BRANCH=master; MACHINE_NAME=smithi; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH -n 7 -m $MACHINE_NAME -s perf-basic -k distro -e $CEPH_QA_EMAIL -N 3
-09 03 * * 6   CEPH_BRANCH=master; MACHINE_NAME=smithi; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH -n 7 -m $MACHINE_NAME -s powercycle -k distro -e $CEPH_QA_EMAIL
+09 03 * * 4,6   CEPH_BRANCH=master; MACHINE_NAME=smithi; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH -n 7 -m $MACHINE_NAME -s powercycle --subset 1/4 -k distro -e $CEPH_QA_EMAIL
```

Comment:
should cycle through subsets like we do for rados

Reply:
no, i think if we just run as is that'd be good
disagree ?

Reply:
if we don't change the numerator, we run exactly the same jobs, and don't get all the combinations

Reply:
See https://github.com/ceph/ceph/pull/35077